addons/website/static/src/scss/bootstrap_overridden.scss

```diff
@@ -279,3 +287,40 @@ $alert-border-scale: -30% !default;
 
 // HR Separator
 $hr-color: $border-color !default;
+
+// Pagination
+
+// Arbitrary values to get a circle for 1 digit numbers
+$pagination-padding-y: 0.4ch !default;
+$pagination-padding-x: 1.2ch !default;
+
+$pagination-border-radius: $border-radius-pill !default;
+$pagination-margin-start: map-get($map: $spacers, $key: 1) !default;
+$pagination-border-color: transparent !default;
+$pagination-bg: $body-bg !default;
+$pagination-color: color-contrast($pagination-bg) !default;
+$pagination-transition: none !default;
+
+$pagination-active-bg: tint-color(o-color('o-cc4-bg'), 80%) !default;
+$pagination-active-color: increase-contrast(o-color('o-cc4-bg') , $pagination-active-bg) !default;
```

Comment:
I'm not sure to understand the issue, could you explain ?
As pagination is a system UI element, it makes sense that its color follows to the website’s theme colors 🤔

Reply:
The idea is indeed to have  `cc4` "related" to the other colors :) 

Like for the[ previous answer ](https://github.com/odoo/odoo/pull/120302#discussion_r1347577543), these changes ultimately make the ui more consistent and its edition predictable.

Reply:
> As pagination is a system UI element, it makes sense that its color follows to the website’s theme colors 🤔

When you are saying it like that of course :wink:

> Like for thehttps://github.com/odoo/odoo/pull/120302#discussion_r1347577543, these changes ultimately make the ui more consistent and its edition predictable.

I think it is very much different from the light/dark debate. While you can **hope** the user will cc2 light and cc5 dark so that light/dark used everywhere on the website works, we cannot expect them to keep a consistent color for cc4 that would work for a random element like the pagination. In the end, I don't really understand the need for the pagination element to have a different color than the "active" state of button / the primary color.

You are saying this is making things predictable but it's very much the opposite to me: you have no way to understand that changing this;
![image](https://github.com/odoo/odoo/assets/10338094/98fc446c-fb13-4004-ac41-5ced16fb6a91)
would change the pagination in your eCommerce :shrug: 

Note: for the light and dark, I am thinking the same actually: it should be about color-2 and color-5 if we do it... not cc2-bg and cc5-bg... no?

Reply:
Conclusion of the IRL talk: we'll keep pagination linked to the standard active color instead of being linked to a preset's background. Same idea as https://github.com/odoo/odoo/pull/120302#discussion_r1348867146